src/examples/src/crud/App/composition.js

```diff
@@ -2,7 +2,7 @@ import { ref, reactive, computed, watch } from 'vue'
 
 export default {
   setup() {
-    const names = reactive(['Emil, Hans', 'Mustermann, Max', 'Tisch, Roman'])
+    const names = reactive(['Шевченко, Ірина', 'Матринчук, Сергій', 'Федоренко, Роман'])
```

Comment:
```suggestion
    const names = reactive(['Шевченко, Ірина', 'Матрунчик, Сергій', 'Федоренко, Роман'])
```

якщо це про мене малось на увазі)))